gotham/src/middleware/cookie/mod.rs

```diff
@@ -24,6 +24,7 @@ impl CookieParser {
             .get_all(COOKIE)
             .iter()
             .flat_map(HeaderValue::to_str)
+            .flat_map(|cs| cs.split("; "))
```

Comment:
It would seem safer to split on ";" and then `trim`. I guess most browsers insert a space, but I'm not sure it's technically required.